app/backend/prepdocs.py

```diff
@@ -333,42 +261,32 @@ async def main(strategy: Strategy, setup_index: bool = True):
         action="store_true",
         help="Remove all blobs from blob storage and documents from the search index",
     )
+
+    # Optional key specification:
     parser.add_argument(
-        "--localpdfparser",
-        action="store_true",
-        help="Use PyPdf local PDF parser (supports only digital PDFs) instead of Azure Document Intelligence service to extract text, tables and layout from the documents",
-    )
-    parser.add_argument(
-        "--localhtmlparser",
-        action="store_true",
-        help="Use Beautiful soap local HTML parser instead of Azure Document Intelligence service to extract text, tables and layout from the documents",
-    )
-    parser.add_argument(
-        "--documentintelligenceservice",
+        "--searchkey",
         required=False,
-        help="Optional. Name of the Azure Document Intelligence service which will be used to extract text, tables and layout from the documents (must exist already)",
+        help="Optional. Use this Azure AI Search account key instead of the current user identity to login (use az login to set current user for Azure)",
     )
     parser.add_argument(
-        "--documentintelligencekey",
+        "--storagekey",
         required=False,
-        help="Optional. Use this Azure Document Intelligence account key instead of the current user identity to login (use az login to set current user for Azure)",
+        help="Optional. Use this Azure Blob Storage account key instead of the current user identity to login (use az login to set current user for Azure)",
     )
     parser.add_argument(
-        "--searchimages",
-        action="store_true",
-        required=False,
-        help="Optional. Generate image embeddings to enable each page to be searched as an image",
+        "--datalakekey", required=False, help="Optional. Use this key when authenticating to Azure Data Lake Gen2"
     )
     parser.add_argument(
-        "--visionendpoint",
+        "--documentintelligencekey",
         required=False,
-        help="Optional, required if --searchimages is specified. Endpoint of Azure AI Vision service to use when embedding images.",
+        help="Optional. Use this Azure Document Intelligence account key instead of the current user identity to login (use az login to set current user for Azure)",
     )
     parser.add_argument(
-        "--useintvectorization",
+        "--searchserviceassignedid",
```

Comment:
I cannot find ANY evidence of us telling folks how to use this, or even actually using it. Do we even need this arg??

Reply:
yeah, let's create a follow-up pr to remove this